### PR TITLE
Bug 1167525 - [Stingray][Home] Hide home cards when home is not visible

### PR DESCRIPTION
**Pull request by sean2449**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4b766662de2012f899e0a560173602aec88ac3d6

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -162,10 +162,12 @@
     },
 
     onVisibilityChange: function() {
+      this.cardListElem.classList.add('hidden');
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: Shouldn't we put this in `else` clause? It's pretty weird be here.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a8607223ae47f6b8b41fc659d89f9b12cb21b1c9

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=91ca8a013205fdc5f8efb6be3b4b4e6a9d03e77c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=34f63a804e3c73bce9e2e64692a90b6f920c3cbe

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=30e2514cc8803b41b0502dc4de58115c344e9f8b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=718ff8fc0c7c997a3f367fdd94ab78aa40c7bceb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=65c02398e8b020227e12e5a01f561ffca643454f

**Pull request merged**